examples/src/main/java/org/apache/spark/examples/ml/JavaMaxAbsScalerExample.java

```diff
@@ -34,10 +44,17 @@ public static void main(String[] args) {
       .getOrCreate();
 
     // $example on$
-    Dataset<Row> dataFrame = spark
-      .read()
-      .format("libsvm")
-      .load("data/mllib/sample_libsvm_data.txt");
+    List<Row> data = Arrays.asList(
```

Comment:
The data in the file is fine, but uses sparse vectors so that when the result is output, it doesn't really show anything.  Using just a small sample dataset, you can see what it is doing from the output

before
```
+-----+--------------------+--------------------+
|label|            features|      scaledFeatures|
+-----+--------------------+--------------------+
|  0.0|(692,[127,128,129...|(692,[127,128,129...|
|  1.0|(692,[158,159,160...|(692,[158,159,160...|
|  1.0|(692,[124,125,126...|(692,[124,125,126...|
```
after
```
+--------------+----------------+                                               
|      features|  scaledFeatures|
+--------------+----------------+
|[1.0,0.1,-8.0]|[0.25,0.01,-1.0]|
|[2.0,1.0,-4.0]|  [0.5,0.1,-0.5]|
|[4.0,10.0,8.0]|   [1.0,1.0,1.0]|
```